src/seraphis_core/legacy_decoy_selector_flat.h

```diff
@@ -47,29 +47,37 @@ namespace sp
 
 ////
 // LegacyDecoySelectorFlat
-// - get a set of unique legacy ring members, selected from a flat distribution across the range of available enotes
+// - get a set of unique legacy ring members, selected from a flat distribution across the range of available
+//   enotes with the same ledger indexing amount
 ///
 class LegacyDecoySelectorFlat final : public LegacyDecoySelector
 {
 public:
+//member types
+    ///                                       ledger amount  ->         ( min index   ,   max index  )
```

Comment:
```suggestion
    /// [ ledger amount : {min index, max index} ]
```
I like to do this syntax for maps. You don't have to adopt it.